### PR TITLE
Adding language preference page 186713028

### PR DESCRIPTION
**Pull request by sree-cfa**
Adding language preference page 

PT#186713028

### CODE REVIEW

File: src/main/resources/templates/gcc/onboarding-language-pref.html [hunk 1]
```diff
@@ -1,18 +1,38 @@
...
+        <th:block th:replace="~{fragments/icons :: languagePreferences}"></th:block>
+        <th:block
+            th:replace="~{fragments/cardHeader :: cardHeader(header=#{onboarding-language-pref.header})}"/>
+        <th:block th:replace="~{'fragments/form' :: form(action=${formAction}, content=~{::formContents})}">
+          <th:block th:ref="formContents">
             <div class="form-card__content">
-                <!-- Put inputs here -->
+              <th:block
+                  th:replace="~{fragments/inputs/select :: select(label=#{onboarding-language-pref.language-to-read-label}, inputName='languageRead', helpText=#{onboarding-language-pref.language-to-read-help-text}, content=~{::content})}">
+                <th:block th:ref="content">
```
Comment (coltborg) [inline src/main/resources/templates/gcc/onboarding-language-pref.html:19]: These `th:ref` have to be unique. Rn they're double rendering:

2024-02-07 at 16 01 10@2x

**Review by enyia21 — APPROVED** [review]

File: src/main/resources/templates/gcc/onboarding-language-pref.html [hunk 2]
```diff
@@ -1,18 +1,38 @@
...
 <div class="page-wrapper">
   <div th:replace="~{fragments/toolbar :: toolbar}"></div>
   <section class="slab">
     <div class="grid">
       <div th:replace="~{fragments/goBack :: goBackLink}"></div>
       <main id="content" role="main" class="form-card spacing-above-35">
-        <th:block th:replace="~{fragments/cardHeader :: cardHeader(header=#{placeholder}, subtext=#{placeholder})}"/>
-        <th:block th:replace="~{fragments/form :: form(action=${formAction}, content=~{::formContent})}">
-          <th:block th:ref="formContent">
+        <th:block th:replace="~{fragments/icons :: languagePreferences}"></th:block>
```
Comment (coltborg) [inline src/main/resources/templates/gcc/onboarding-language-pref.html:11]: I think we'll need to create a new icon for us in `gcc-icons` as as the colors of the icon are different to the icon in the library:

2024-02-07 at 17 17 19@2x

**Comment by coltborg** [comment]
There are some [acceptance criteria](https://www.pivotaltracker.com/story/show/186713028) for this screen that aren't happening: 

> * When the page first loads, the default language for both reading and writing, and speaking should be set to the current language of the application.
> * If a client changes the language that they prefer to read and write to a different language than the current language of the application, then site language should be immediately set to the selected language.
> * Tracking: Once the onboarding-language-pref screen is completed, the pref_written_language and pref_spoken_language properties should be set with the client's chosen language and applied to all future events.

You could either include them in the PR or accept this PR and continue the work in another PR.

**Review by coltborg — APPROVED** [review]
lgtm;

**Pull request merged**